WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationStepView.swift

```diff
@@ -17,17 +17,14 @@ class MigrationStepView: UIView {
     private let centerView: UIView
     private let actionsView: MigrationActionsView
 
-    private lazy var centerContentView: UIView = {
-        let view = UIView()
-        view.addSubview(centerView)
-        return view
-    }()
-
     private lazy var mainStackView: UIStackView = {
-        let stackView = UIStackView(arrangedSubviews: [headerView, centerContentView])
+        let stackView = UIStackView(arrangedSubviews: [headerView, centerView, UIView()])
```

Comment:
The added `UIView()` is to make the `centerView` appear centered in the `UIStackView`.